src/backend/src/controllers/description-Bullets.controllers.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from 'express';
```

Comment:
pls rename this file to be kebab case (`description-bullets.controllers.ts`) you might have to use `git mv` though becuase sometimes git doesn't recognize just changing cases in file names